data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1226 @@
+			choice
+				`	"Maybe it's better I do leave them alone."`
+					goto "better"
+				`	"There must be something I can do."`
+					goto "understand"
+
+			label "unsure"
+			action
+				set "gegno word: unsure"
+			`	Adrauni turns to you briefly, staring at you blankly, then turns to the Gegno and mutters something in their language before responding to you. "And why is it that you are unsure?"`
+			choice
+				`	"I wasn't sure you could convince them."`
+					goto "convince"
+				`	"I don't know anything about them."`
+
+			action
+				set "gegno word: friend"
+			`	After a pause, Adrauni turns toward you, stares at you briefly, then looks back forward as the three of you continue through the ring. "Quite an important conclusion, is that not? Many times I have said this, that the Gegno are not what I would consider humans to posses knowledge of. Should you learn more, I'm afraid even then the Gegno do not contrastingly desire, what was the human word - camaraderie?" The Gegno then asks something in their language, and Adrauni begins a somewhat lengthy conversation with them.`
+			choice
+				`	"Maybe it's better I do leave them alone."`
+					goto "better"
+				`	"There must be something I can do."`
+					goto "understand"
+
+			label "interest"
+			`	The Quarg turns back to you. "You are here not at the whim of your fanatic species, but at your own behest. Actions you take may be that of a human, but are not willed by that of humanity." Blinking slowly, Adrauni then turns to the Gegno and begins a somewhat lengthy conversation with them.`
+				goto "visit a"
+
+			label "understand"
+			`	You walk in silence for an awkward period of time. The strange architecture of the ringworld does little to comfort you, as it feels like you've walked down the same path several times now. A few moments later, you eventually begin following Adrauni and the Gegno down what appears to be a different section, though it is strangely wider than previous ones. Before you begin asking more questions, Adrauni finally replies. "It is apparent that you may understand something of your fragile circumstance. It is no longer what I, rather, the Quarg, can do for you. It is what you can do for yourself."`
+				goto "visit a"
+
+			label "better"
+			`	Adrauni speaks to the Gegno once more, this time for an extended period of time. The strange architecture of the ringworld does little to comfort you, as it feels like you've walked down the same path several times now. A few moments later, you eventually begin following Adrauni and the Gegno down what appears to be a different section, though it is strangely wider than previous ones. "Yes, it may be that your timely departure is the most favorable outcome in this affair." A long, awkward silence ensues, at the end of which Adrauni briefly comes to a hault, still facing forward, now crossing its armos behind its back. "Though you and I figure that is - unlikely."`
```

Comment:
```suggestion
			`	Adrauni speaks to the Gegno once more, this time for an extended period of time. The strange architecture of the ringworld does little to comfort you, as it feels like you've walked down the same path several times now. A few moments later, you eventually begin following Adrauni and the Gegno down what appears to be a different section, though it is strangely wider than previous ones. "Yes, it may be that your timely departure is the most favorable outcome in this affair." A long, awkward silence ensues, at the end of which Adrauni briefly comes to a halt, still facing forward, now crossing its arms behind its back. "Though you and I figure that is - unlikely."`
```
crossing its `armos` lmao

Reply:
`"Though you and I figure that is - unlikely."`
What does this mean in Quargspeak? I had trouble understanding this one.